doc/ref_model/chapters/chapter05.md

```diff
@@ -181,7 +181,7 @@ This section will detail NFVI SW profiles and associated configurations for the
 | .conf | Feature | Type  | Basic | Network Intensive | Compute Intensive |
 |------------------|----------------|----------------|----------------|----------------|----------------|
 | nfvi.net.acc.cfg.001 | vSwitch optimisation | YeS/No and SW Optimisation | N | Y, DPDK | Y, DPDK |
-| nfvi.net.acc.cfg.002 | Support of HW offload | YeS/No | N | Y, support of SR-IOV and  SmartNic |Y, support of SR-IOV and  SmartNic |
+| nfvi.net.acc.cfg.002 | Support of HW offload | YeS/No | N | Y, support of SmartNic |Y, support of SmartNic |
```

Comment:
Since HW offload is required for Network Intensive and Compute intensive, what are the options in the absence of SmartNICs?

Reply:
SRIOV is not a HW offload technology it is a mechanism to expose a hardware device into multiple ones. A other example of hardware offload is a HW acceleration card.

Reply:
@pgoyal01 , first why HW offload is required for Network Intensive and Compute intensive?
as per my previous response and to make more clear 

for Compute Intensive (ch02: for VNFCs that require predictable computing performance and low network latency ) : from practical speaking no need to have HW offload as we can achieve this profile by OVS or OVS+DPDK with minimal vCPU allocated to PMDs

for Network Intensive: (high network throughput and low network latency.) which required one of the below Tech :
1-SRIOV
2- OVS+DPDK (a large number of Core be allocated to PMDs )
3- HW offloading cards  (Marketing name SmartNIC ) or FPGA to offload the allocated core on the NIC card 

So the above 3 option is available and Yes I admit that SRIOV is proved technology for high performance but it has many drawbacks and I  considered it as fake virtualization and against what we looking for,

So my recommendation to remove it totally.
I hope it is clear 

@markshostak @rabi-abdel @karinesevilla @petorre